sql/03_create-view.sql

```diff
@@ -0,0 +1,38 @@
+-- sql/03_create-view.sql
+
+DROP VIEW IF EXISTS api.events_view;
+CREATE OR REPLACE VIEW api.events_view AS
+SELECT
+  *,  
+  -- flatten user_id: pick first array element, then scalar field, then fallback
+  COALESCE(
+    event->'subject'->>0,    -- dhair2-style array
+    event->>'subject',       -- simple string
+    event->>'user_id'        -- legacy key
+  ) AS user_id,
+
+  -- flatten event_type: use explicit action first, then event_type, then tags, then message
+  COALESCE(
+    event->>'action',
+    event->>'event_type',
+    event->'tags'->>0,
+    event->>'message'
+  ) AS event_type,
+
+  -- schema_version: LTT first, then version, else legacy
+  CASE
+    WHEN event->>'source_system' = 'LTT'   THEN 'ltt'
```

Comment:
```suggestion
```
Unless I'm missing something, it would probably better to remove this specific case for LTT. We're not so concerned with each individual project/system, and LTT events *should* have a version